### ISSUE TITLE
Improve readme usage section

### ISSUE THREAD

**Issue by dbrakenhoff** [pastas/pastastore#53]
Currently contains too much technical info about Connectors. Should contain something along these lines:

Gettings started.
Define Connector [link to extra info in docs] and PastaStore:

```python
conn = pst.PasConnector("test", "./pastas_db")
pstore = pst.PastaStore(conn.name, conn)
```

Add head observation time series:
```python
o = pd.read_csv("obs.csv", index_col=[0])
meta = {"x": 100, "y": 200}
pstore.add_oseries(o, "obs1", metadata=meta)
# view oseries metadata with
pstore.oseries
```

Add stresses time series:
```python
p = pd.read_csv("prec.csv", index_col=[0]
e = pd.read_csv("evap.csv", index_col=[0]
pstore.add_stress(p, "prec1", kind="prec", metadata={"x":110, "y":195}
pstore.add_stress(e, "evap1", kind="evap", metadata={"x":110, "y":195}
# view stresses metadata
pstore.stresses
```
Create time series models
```python
ml = pstore.create_model("obs1", add_recharge=True)
ml.solve()
pstore.add_model(ml)
```

Load model from PastaStore:
```python
ml2 = pstore.get_models("obs1")
```

**Comment by dbrakenhoff** [comment]
This has been improved on dev, leaving this open until I manage to also implement these improvements in the docs.

**Comment by dbrakenhoff** [comment]
Closed by #80.
